Day-II-Calculator/calculator/src/components/ButtonComponents/ActionContainer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './Button.css';
+import ActionButton from './ActionButton';
+
+const ActionContainer = (props) => {
+    return (
+        <div class='action-buttons'>
+            <div class='action'>
+                <ActionButton class='buttonStyle' text='/' />
+            </div>
+            <div class='action'>
+                <ActionButton class='buttonStyle' text='x' />
+            </div>
+            <div class='action'>
+                <ActionButton class='buttonStyle' text='-' />
+            </div>
+            <div class='action'>
+                <ActionButton class='buttonStyle' text='+' />
+            </div>
+            <div class='action'>
+                <ActionButton class='buttonStyle' text='=' />
+            </div>
```

Comment:
see any repetition here?

(also, don't need to put every ActionButton in its own `div`)